README.md

```diff
@@ -1,12 +1,155 @@
 
 `NB!` current implementation supports only [`hivemetastore`](conf/nomad/hive.hcl#L99)
+
+## Example usage
+The example-code shows the minimum of what you need do to set up this module.
+```hcl-terraform
+module "minio" {
+  source = "github.com/fredrikhgrelland/terraform-nomad-minio.git?ref=0.1.0"
+
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+
+  # minio
+  service_name                    = "minio"
+  host                            = "127.0.0.1"
+  port                            = 9000
+  container_image                 = "minio/minio:latest"
+  access_key                      = "minio"
+  secret_key                      = "minio123"
+  buckets                         = ["default", "hive"]
+  container_environment_variables = ["JUST_EXAMPLE_VAR1=some-value", "ANOTHER_EXAMPLE2=some-other-value"]
+
+  # mc
+  mc_service_name                    = "mc"
+  mc_container_image                 = "minio/mc:latest"
+  mc_container_environment_variables = ["JUST_EXAMPLE_VAR3=some-value", "ANOTHER_EXAMPLE4=some-other-value"]
+}
+
+module "postgres" {
+  source = "github.com/fredrikhgrelland/terraform-nomad-postgres.git?ref=0.1.0"
+
+  # nomad
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+
+  # postgres
+  service_name                    = "postgres"
+  container_image                 = "postgres:12-alpine"
+  container_port                  = 5432
+  admin_user                      = "hive"
+  admin_password                  = "hive"
+  database                        = "metastore"
+  container_environment_variables = ["PGDATA=/var/lib/postgresql/data"]
+}
+
+module "hive" {
+  source = "./.."
+
+  # nomad
+  nomad_datacenters      = ["dc1"]
+  nomad_namespace        = "default"
+  nomad_job_switch_local = false
+
+  # hive
+  hive_service_name                    = "hive-metastore"
+  hive_container_port                  = 9083
+  hive_docker_image                    = "fredrikhgrelland/hive:3.1.0"
+  hive_container_environment_variables = ["SOME_EXAMPLE=example-value"]
+
+  # hive - minio
+  hive_bucket = {
+    default     = "default",
+    hive        = "hive"
+  }
+  minio_service = {
+    service_name = module.minio.minio_service_name,
+    port         = 9000,
+    access_key   = module.minio.minio_access_key,
+    secret_key   = module.minio.minio_secret_key,
+  }
+
+  # hive - postgres
+  postgres_service = {
+    service_name  = module.postgres.service_name
+    port          = module.postgres.port
+    database_name = module.postgres.database_name
+    username      = module.postgres.username
+    password      = module.postgres.password
+  }
+
+  depends_on = [
+    module.minio,
+    module.postgres
+  ]
+}
+```
+
+### Verifying setup
+You can verify the setup by connection to minio with a consul proxy. Check out the [required software](#required-software) section.
+```sh
+make proxy-to-minio
+```
+Minio should be available at [localhost:9000](http://localhost:9000/minio/).
```

Comment:
I think this part should be different. How to verify hive setup

Reply:
Yes, you are probably right. Do you have any suggestions?

Reply:
You need query tables via beeline and consul proxy. Hint here https://github.com/zhenik-poc/big-data-stack-practice#useful-commands-debug

Reply:
Alright, thanks!